addons/website/models/website_visitor.py

```diff
@@ -37,17 +37,17 @@ class WebsiteVisitor(models.Model):
     country_flag = fields.Binary(related="country_id.image", string="Country Flag")
     lang_id = fields.Many2one('res.lang', string='Language', help="Language from the website when visitor has been created")
     visit_count = fields.Integer('Number of visits', default=1, readonly=True, help="A new visit is considered if last connection was more than 8 hours ago.")
-    visitor_page_ids = fields.One2many('website.visitor.page', 'visitor_id', string='Visited Pages History', readonly=True)
+    website_track_ids = fields.One2many('website.track', 'visitor_id', string='Visited Pages History', readonly=True)
     visitor_page_count = fields.Integer('Page Views', compute="_compute_page_statistics")
     page_ids = fields.Many2many('website.page', string="Visited Pages", compute="_compute_page_statistics", store=True)
     page_count = fields.Integer('# Visited Pages', compute="_compute_page_statistics")
     time_since_last_action = fields.Char('Last action', compute="_compute_time_statistics", help='Time since last page view. E.g.: 2 minutes ago')
     is_connected = fields.Boolean('Is connected ?', compute='_compute_time_statistics', help='A visitor is considered as connected if his last page view was within the last 5 minutes.')
```

Comment:
> A visitor is considered as connected if his last page view was within the last 5 minutes.

Really strange for end user. Because, it is only true for tracked item !
Maybe we should have a ternary status: yes (<5 min), maybe (<30m), probably not (>30m)